iis/mymodule.cpp

```diff
@@ -800,17 +755,6 @@ CMyHttpModule::OnBeginRequest(
         goto Finished;
 	}
 
-    // Get request body without holding lock, because some clients may be slow at sending
-    LeaveCriticalSection(&m_csLock);
```

Comment:
I remember previously we added this for remove global lock when body request was sent slowly,  since we are removing this code, do we still have the same issue? Or it could be resolved by the concurrency fix?

Reply:
We have removed the locking around modsecProcessRequest, so when modsecProcessRequest calls ReadBodyCallback which might block if the client is sending slow, it is no longer holding the lock.